app/api/resources/mentorship_relation.py

```diff
@@ -325,12 +325,12 @@ def put(cls, request_id):
         # TODO check if user id is well parsed, if it is an integer
 
         user_id = get_jwt_identity()
-        response = DAO.cancel_relation(user_id=user_id, relation_id=request_id)
+        response = DAO.cancel_relation(user_id=user_id, relation_id=relation_id)
 
         return response
 
 
-@mentorship_relation_ns.route("mentorship_relation/<int:request_id>")
+@mentorship_relation_ns.route("mentorship_relation/<int:relation_id>")
```

Comment:
No need to do any of the changes(i.e from request_id to relation_id) in this class. Since, we still haven't accepted the relation request.